pkg/kvstore/consul.go

```diff
@@ -305,6 +306,7 @@ func (c *consulClient) LockPath(ctx context.Context, path string) (KVLocker, err
 
 // watch starts watching for changes in a prefix
 func (c *consulClient) watch(ctx context.Context, w *Watcher) {
+	scope := GetScopeFromKey(strings.TrimRight(w.Prefix, "/"))
```

Comment:
> We additionally trim a possible
leading / separator from the prefix, to ensure that it doesn't count
against the number of tokens for scope computation. Most notably, this
allows to coalescence the kvstore_events_queue metrics for the synced
prefixes of all clusters into a single set, which does not depend on the
number of clusters we are connected to.

Maybe it is worth adding some small information in upgraded metrics in upgrade docs?